src/atom.cpp

```diff
@@ -2062,28 +2062,63 @@ void Atom::add_callback(int flag)
   for (ifix = 0; ifix < modify->nfix; ifix++)
     if (modify->fix[ifix] == NULL) break;
 
-  // add callback to lists, reallocating if necessary
+  // insert callback into sorted lists, reallocating if necessary
```

Comment:
please move this comment above the fix for loop,
since it applies to all 3 lists

Reply:
sorry, I mis-read the code for this comment.  Just ignore!